### PR TITLE
Adjust dev setup to match solo ci

### PR DESCRIPTION
**Pull request by brenzi**
* make Alice sole councillor
* endow Bob

to make bootstrapping and community growth work with https://github.com/encointer/encointer-node/pull/197

### CODE REVIEW

File: polkadot-parachains/src/chain_spec_helpers.rs [hunk 1]
```diff
@@ -48,21 +48,15 @@ pub struct WellKnownKeys;
 
 impl WellKnownKeys {
 	pub fn endowed() -> Vec<AccountId> {
-		Self::council()
+		vec![Alice.to_account_id(), Bob.to_account_id()]
```
Comment (clangenb) [inline polkadot-parachains/src/chain_spec_helpers.rs:51]: Can we not also endow `Charlie` so we can remove his fauceting to save some time in the `bootstrap_demo_community.py`? Fauceting is anyhow tested in the `bot-community.py`

Comment (brenzi) [reply polkadot-parachains/src/chain_spec_helpers.rs:51]: we could but I just wanted the config to match the encointer-node. I actually like that the bootstrap_demo_community tests as much basic stuff as possible already

Comment (brenzi) [reply polkadot-parachains/src/chain_spec_helpers.rs:51]: not entirely sure

Comment (brenzi) [reply polkadot-parachains/src/chain_spec_helpers.rs:51]: ok, will add Charlie ;-)

**Pull request merged**